app/services/live_lookup.rb

```diff
@@ -1,19 +1,21 @@
 # LiveLookup serves as a wrapper around specific ILS implementations
-#   for requesting live information about item availability.
+# for requesting live information about item availability.
 class LiveLookup
   delegate :as_json, :to_json, to: :records
 
   def initialize(ids)
     @ids = [ids].flatten.compact
   end
 
-  # Uses either the Folio or Sirsi LiveLookup service
-  # depending on whether FOLIO_LIVE_LOOKUP is set.
+  # Uses the LiveLookup service specified by Settings.LIVE_LOOKUP_SERVICE,
+  # or if nothing is configured fall back to LiveLookup::Sirsi
   def records
-    if Settings.FOLIO_LIVE_LOOKUP
-      LiveLookup::Folio.new(@ids).records
-    else
-      LiveLookup::Sirsi.new(@ids).records
-    end
+    live_lookup_service.new(@ids).records
+  end
+
+  private
+
+  def live_lookup_service
+    (Settings.LIVE_LOOKUP_SERVICE || 'LiveLookup::Sirsi').constantize
```

Comment:
Can this be downcase?  This makes it look like a constant, but it's actually a method.

Reply:
Also, can you put `live_lookup_service: LiveLookup::Sirsi` in `config/settings.yml` rather than a default here?

Reply:
Sounds good. Updated with both suggestions.